pulsar-client-admin-api/src/main/java/org/apache/pulsar/client/admin/Namespaces.java

```diff
@@ -2061,10 +2062,11 @@ void setBookieAffinityGroup(String namespace, BookieAffinityGroupData bookieAffi
      * @param bundle range of bundle to split
      * @param unloadSplitBundles
      * @param splitAlgorithmName
+     * @param splitBoundaries
      * @throws PulsarAdminException
      */
-    void splitNamespaceBundle(String namespace, String bundle, boolean unloadSplitBundles, String splitAlgorithmName)
-            throws PulsarAdminException;
+    void splitNamespaceBundle(String namespace, String bundle, boolean unloadSplitBundles,
```

Comment:
We can't change the API directly which will introduce API breaking change.

Reply:
add two new APIs for `Namespaces` and keep the old API safe